packages/neuron-ui/src/locales/fr.json

```diff
@@ -0,0 +1,1191 @@
+        "lock-script": "Script de verrouillage",
+        "lock-script-title": "Informations sur l'adresse",
+        "deprecated-address-format": "Adresse au format obsolète"
+      },
+      "addresses": {
+        "title": "Carnet d'adresses",
+        "addresses": "Adresses",
+        "type": "Type",
+        "address": "Adresse",
+        "identifier": "Identifiant",
+        "description": "Description",
+        "balance": "Solde",
+        "transactions": "TXs",
+        "receiving-address": "Réception",
+        "change-address": "Changement",
+        "copy-address": "Copier l'adresse",
+        "request-payment": "Demander un paiement",
+        "view-on-explorer": "Voir sur l'explorateur",
+        "default-description": "Aucun",
+        "all-address": "Toutes"
+    },
+    "settings":{
+      "title": {
+        "normal": "Paramètres",
+        "mac": "Préférences"
+      },
+      "go-to-overview": "Aller à l'aperçu",
+      "setting-tabs": {
+        "general": "Général",
+        "wallets": "Wallets",
+        "network": "Réseau",
+        "data": "Data"
+      },
+      "general": {
+        "show": "Afficher",
+        "hide": "Masquer",
+        "version": "Version",
+        "language": "Langue",
+        "select-language": "Sélectionner la langue",
+        "apply": "Appliquer"
+      },
+      "wallet-manager": {
+        "edit-wallet": {
+          "wallet-name": "Nom du Wallet",
+          "password": "Mot de passe",
+          "new-password": "Nouveau mot de passe",
+          "confirm-password": "Confirmer le mot de passe",
+          "edit-wallet": "Éditer le Wallet"
+        },
+        "edit-success": "Informations du Wallet mises à jour avec succès",
+        "delete-wallet-title": "Veuillez entrer le mot de passe de {{name}}",
+        "password": "Mot de passe",
+        "wallet-detail": {
+          "balance": "Solde"
+        }
+      },
+      "network": {
+       "online": "En ligne",
+       "offline": "Hors ligne",
+       "add-network": "Ajouter un réseau",
+       "remove-network": "Supprimer le réseau",
+       "remove-network-msg": "Êtes-vous sûr de vouloir supprimer le réseau sélectionné ?",
+        "edit-network": {
+         "title": "Modifier le réseau",
+         "rpc-url": "URL RPC",
+         "name": "Nom du réseau",
+         "input-rpc": "Veuillez entrer rpc",
+         "input-network": "Veuillez entrer le nom" 
+        },
+        "edit-success": "Modification réussie du réseau",
+        "mainnet": "Mainnet",
+        "testnet": "Testnet",
+        "lightTestnet": "Testnet léger",
+        "lightMainnet": "Mainnet léger",
+        "devnet": "Devnet",
+        "switch-network-type": "Basculer vers {{type}}"
+      },  
+        "locale": {
+        "en": "Anglais",
+        "en-US": "Anglais (États-Unis)",
+        "zh": "中文(简体)",
+        "zh-TW": "中文(繁體)",
+        "fr": "Français"
+      },
+      "data": {
+        "ckb-node-data": "Configuration & stockage du noeud CKB",
```

Comment:
We should note if the layout is impacted by this sentence because the length increases much @Keith-CY